internal/api/action_handler.go

```
@@ -868,7 +864,13 @@ func (h *ActionHandler) DispatchAction(ctx context.Context, req *connect.Request
 		}
 		timeoutSeconds = action.TimeoutSeconds
 		actionID = &source.ActionId
+		// These values are currently unconditionally overwritten by the
+		// re-sign call below — see issue #137 for the contract decision
+		// pending. Marked ignored so the new staticcheck CI step passes
+		// without papering over the underlying ambiguity.
+		//lint:ignore SA4006 see https://github.com/manchtools/power-manage-server/issues/137
 		signature = action.Signature
```

golangci-lint (2.12.1)

[error] ineffectual assignment to signature

(ineffassign)